doc/src/learn/transactions.md

```diff
@@ -63,6 +63,27 @@ This transaction type combines several coin objects into one. It includes the fo
 
 Inputs: A list of unique object references pointing to mutable objects owned by the sender. The objects must all have the same type: `sui::coin::Coin<T>` with arbitrary `T`--that is, any fungible token. The list must contain at least two objects. All objects except the first one will be destroyed, and the new value of the first object will be its old value plus the sum of the value of all destroyed objects. The gas input object from above cannot also appear as an object input.
 
+## Transactions flow - example
+
+Here's an example showing how objects and transactions are connected to each other in Sui.
+
+In the example below we have two objects:
+- object A with 5 SUI coins that belongs to Tom
+- object B with 2 SUI coins that belongs to John
+
+Tom decided to send 1 SUI coin to Alice. In this case the object A will be the input to this transaction and 1 SUI coin will be debited from this object. The output of the transaction will be two objects: 
+- object A with 4 SUI coins which is still assigned to Tom
+- new created object B with 1 SUI coin that belongs now to Alice
```

Comment:
```suggestion
- new created object B with 1 SUI coin that belongs now to Alice
```
There is already an Object B with 2 SUI that belongs to John, do we need Object C here?

Reply:
yes, it should be object C